design/authn_azure/authn_azure_solution_design.md

```diff
@@ -420,44 +420,44 @@ Azure authenticator performance should conform with our other authenticators wit
-| 15 | `xms_mirid` field of the Azure token is missing                                                                             | VM sends authentication request to Conjur with invalid Azure token  | Return 400 Bad Request error and `TokenFieldNotFoundOrEmpty` message appears in log      | Integration               | [ ]            |
-| 16 | One or more fields in the Azure token don't match the Conjur host annotations                                               | VM sends authentication request to Conjur with valid Azure token    | Return 401 Unauthorized error and `InvalidApplicationIdentity` message appears in log    | Integration               | [ ]            |
+|    | Given                                                                                                                       | When                                                                | Then                                                                                     | Manual / UT / Integration | Status                       |Core Test  |
+|----|-----------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------|------------------------------------------------------------------------------------------|---------------------------|------------------------------|------------
+| 1  | *Vanilla flow* - a host identity exists for Azure VM and added to privileged group for authentication on webservice         | VM sends authentication request with a valid Azure token            | Conjur access token is returned in body and action is logged                             | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
+| 2  | *Vanilla flow* - a user identity exists for Azure VM and added to privileged group for authentication on webservice         | VM sends authentication request with a valid Azure token            | Conjur access token is returned in body and action is logged                             | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
+| 3  | A host identity for Azure VM has been successfully authorized by Conjur                                                     | VM sends request with valid Azure token to fetch secret             | Azure resource can successfully send GET request for a secret and action is logged       | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
+| 4  | A host identity for Azure VM has not been added to privileged group                                                         | VM sends request with valid Azure token                             | Return 401 Unauthorized error and `RoleNotAuthorizedOnWebservice` and action is logged   | Integration               | <ul><li> [ ] Tested</li></ul>| Yes       |
```

Comment:
This is not a core test in my opinion. It is not an authn-azure flow and is not related to the Azure authenticator flow. This validation is a general one.

Reply:
Done